packages/common/src/schemas/documents/workflow/documents/schemas/UG.ts

```diff
@@ -0,0 +1,140 @@
+import { TDocument } from '@/schemas';
+import { Type } from '@sinclair/typebox';
+
+export const getUgandaDocuments = (): TDocument[] => {
+  const TypeAlphanumericString = Type.String({ pattern: '^[a-zA-Z0-9]*$' });
+  const TypePastDate = Type.String({
+    format: 'date',
+    formatMaximum: new Date().toISOString().split('T')[0], // @TODO: Test this
+  });
+
+  const TypeStringAtLeastOneWord = Type.String({ minLength: 1 });
```

Comment:
Let's extract to some shared schemas file.

Reply:
I don't think these types should be reusable as future cases might want to add additional config properties. I'm okay with setting these types ad-hoc when you need them. In one place "at least one word" can be 2 characters, and in another it could be 1. It's a tough thing managing reuseable code that needs adjustments per usage.
The benefit of making these reusable isn't high enough IMO.
WDYT?